sql/2020/05_Media/image_srcset_candidates.sql

```diff
@@ -0,0 +1,43 @@
+#standardSQL
+# images srcset candidates average
+
+# returns all the data we need from _media
+CREATE TEMPORARY FUNCTION get_media_info(media_string STRING)
+RETURNS STRUCT<
+  num_srcset_all INT64, 
+  num_srcset_candidates_avg INT64
+> LANGUAGE js AS '''
+var result = {};
+try {
+    var media = JSON.parse(media_string);
+
+    if (Array.isArray(media) || typeof media != 'object') return result;
+	
+    result.num_srcset_all = media.num_srcset_all;
+    result.num_srcset_candidates_avg = 
+	    media.num_srcset_all == 0? 0: (media.num_srcset_candidates / media.num_srcset_all);
+
+} catch (e) {}
+return result;
+''';
+
+SELECT
+  client,
+  COUNT(0) AS total_pages,
+  COUNTIF(media_info.num_srcset_all > 0) AS srcset_all,
+  COUNTIF(media_info.num_srcset_candidates_avg >= 1 and media_info.num_srcset_candidates_avg <=3) AS srcset_candidates_1_3,
+  COUNTIF(media_info.num_srcset_candidates_avg >= 1 and media_info.num_srcset_candidates_avg <= 5) AS srcset_candidates_1_5,
+  COUNTIF(media_info.num_srcset_candidates_avg > 5 and media_info.num_srcset_candidates_avg <= 10) AS srcset_candidates_5_10,
+  COUNTIF(media_info.num_srcset_candidates_avg > 10 and media_info.num_srcset_candidates_avg <= 15) AS srcset_candidates_10_15,
+  COUNTIF(media_info.num_srcset_candidates_avg > 15 and media_info.num_srcset_candidates_avg <= 20) AS srcset_candidates_15_20
```

Comment:
Same comments as for `image_alt.sql` above. Should we measure both pages (i.e. at least one result) and total images?